fetch.bs

```diff
@@ -1077,14 +1081,14 @@ run these steps:
  <li><p>Return <var>values</var>.
 </ol>
 
-<p>To determine if a <a>byte sequence</a> <var>value</var> is a
-<dfn>simple range header value</dfn>, perform the following steps. They return a <a>boolean</a>.
+<p>To <dfn lt="parse a single range header value|single range header value">parse a single range header value</dfn>
+from a <a>byte sequence</a> <var>value</var>, run these steps:
```

Comment:
Many parsers indicate the returned value type. I couldn't find a primitive type that would encompass a range type `(start, end)`.

Reply:
It returns failure or a https://infra.spec.whatwg.org/#tuples of two items each of which is null or a number. I'm not sure it's worth writing that out however.